queries/currency.py

```diff
@@ -290,15 +290,17 @@ def sentence(state: QueryStateDict, result: Result) -> None:
                 val = round(val, 2)
             answer = iceformat_float(val)
             response = dict(answer=answer)
-            if gender is None:
-                gender = NON_KVK_CURRENCY_GENDERS.get(target_currency, "kvk")
+            if target_gender is None:
+                target_gender = NON_KVK_CURRENCY_GENDERS.get(target_currency, "kvk")
+            from_gender = NON_KVK_CURRENCY_GENDERS.get(result.currencies[0],'kvk')
             voice_answer = "{0} {1} {2}{3}.".format(
-                result.desc,
+                gssml(result.desc, type='floats', gender=from_gender),
```

Comment:
What's the difference between type "floats" and "float"?

Reply:
`"floats"` means a string containing more than one float, so it uses a regex to find & substitute the numbers. (This suttle difference is maybe a bit confusing though)